tasks/install_and_run_psmdb50.yml

```diff
@@ -0,0 +1,16 @@
+# This task enables Percona Server for MongoDB 50 (psmdb50) main repository, installs and starts modgodb.
+
+# Enable psmdb repo and install psmdb
+  - name: include tasks for enabling psmdb 50 repo
+    include_tasks: ../tasks/enable_psmdb_repo.yml
+    vars:
+      psmdb_repo: "psmdb-50 release"
```

Comment:
the same here, I could just have install_and_run_psmdb.yml, pass the psmdb_repo as variable, one file to rule them all